src/services/game-process-service.js

```diff
@@ -0,0 +1,78 @@
+import axios from 'axios';
+import chalk from 'chalk';
+
+export class GameProcessService {
+    constructor({ gameService, gameSourcesService, pubSubQueueProvider }) {
+        this.paused = false;
```

Comment:
I'm not sure it really makes sense to use pause/resume functionality for this, since it is just reacting to items being added to the queue. The real way to pause this whole pipeline is to pause the poller.